scripts/xsslint/xsslint/linters.py

```diff
@@ -1482,3 +1483,152 @@ def _find_mako_expressions(self, mako_template):
                 start_index = expression.end_index
             expressions.append(expression)
         return expressions
+
+
+class DjangoTemplateLinter(BaseLinter):
```

Comment:
I might have broken `django_linter.py` into its own file and kept the django expressions with its linter since they don't need to be used elsewhere.  It's up to you whether you want to leave as-is.